.gitignore

```diff
@@ -1,2 +1,3 @@
 dist/
 node_modules/
+resources/
```

Comment:
why? I thought `resources` directory is gone

Reply:
Becuz it's still on my local machine